src/components/label/Label.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+import { StyleSheet, View, ViewStyle } from 'react-native';
+import { useTheme } from '../../styles/themes';
+
+import { Body } from '../typography/Body';
+
+type LabelType = 'outlined' | 'filled' | 'soft';
+type LabelColor =
+    | 'error'
+    | 'warning'
+    | 'success'
+    | 'information'
+    | 'secondary'
+    | 'primary'
+    | 'neutral';
+
+interface Props {
+    style?: ViewStyle;
+    text: string;
+    type: LabelType;
+    labelColor?: LabelColor;
+}
+
+export const Label = ({ style, text, type, labelColor = 'neutral' }: Props) => {
+    const theme = useTheme();
+
+    const transparencyValue = '29';
+
+    const getColors = () => {
+        if (labelColor === 'neutral') {
```

Comment:
I don't find that much readable like that. I think I'd rather have an explicit style or something with a set of text/background colors for each status.

Reply:
okay, i'll update that

Reply:
@AudBrou updated!